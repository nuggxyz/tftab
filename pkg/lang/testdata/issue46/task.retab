ben abc {
}

ben def {
}

ben ghi {
}

all_file_yamls = allof("ben", { key= ["label"], regex = "^abc$" })

gen out {
    path = "out/file.yaml"
    data = { afy = [for x in all_file_yamls: x.label] }
}
