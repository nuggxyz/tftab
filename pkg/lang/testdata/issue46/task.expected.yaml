# code generated by retab. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

afy:
    - abc
